components/list-item/src/index.js

```diff
@@ -6,15 +6,16 @@ import {
   LINE_HEIGHT,
   MEDIA_QUERIES,
   NTA_LIGHT,
+  SPACING,
 } from '@govuk-react/constants';
 
 const GListItem = glamorous.li({
-  boxSizing: 'border-box',
   fontFamily: NTA_LIGHT,
```

Comment:
still think i'd rather see the font stuff removed from here and placed into the `OrderedList` and `UnorderedList` components - but not going to block this merge for that

Reply:
I think it should be in both places but not really needed right now

Reply:
had a discussion with @stevesims in the office, I think in summary this is - we are using these components in an environment (DOM) where CSS cascades down. However if we were to have react native compatibility in the future I'm not sure these styles would cascade down. Part of the thinking around CSS in JS is complete encapsulation of the styles for a component so we shouldn't rely on a parent component's styles cascading down.

Reply:
okay i'll take out of `ul`/`ol` - `ListItem` which is the next child from those, has font styling etc

Reply:
okay might just leave as-is for now, if thats okay and if and when we start using react native we can address then. 

I'm not sure having font styling in both the `ul/ol` and the `ListItem` is useful, as the li the only child `ul/ol` can have iirc